docs/react-native/features/network-span-forwarding.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Network Span Forwarding
+sidebar_position: 7
+---
+
+# Network Span Forwarding
+
+For a full explanation of this feature please refer to [Essentials / Product Overview / Network Span Forwarding](/product/network-spans-forwarding/) section.
+
+## Enablement
+
+Once all requirements described in [Network Span Forwarding](/product/network-spans-forwarding/) are met, the feature will be set up by an integrations specialist who will reach out to confirm details.
+
+## Android
+
+You may enable the feature by adding the proper configuration through the `app/src/main/embrace-config.json` placed in the `android` folder that is supposed to be part of your React Native project. NSF is not enabled by default for Android platform for React Native projects. For more information about this file, please refer to the [Configuration File section](/android/features/configuration-file/).
```

Comment:
@jpmunz @bidetofevil do you think it worth to mention something about the need of reaching to us for the remote config being in place? beside the information they already have in here https://embrace.io/docs/product/network-spans-forwarding/ that is mentioned in the first paragraph

Reply:
I do think it's worth a bit of clarification as the distinction between remote/local config probably wouldn't be too obvious to the customer. I'd maybe re-work like this let me know what you think:

Add a last sentence under "Enablement":
> At this point everything should be working on iOS. For Android one additional configuration is required in `android/app/src/main/embrace-config.json`.....

etc. and including what you have under Android there then I think you below you could have:

> ## Turning off the feature in code
> ### iOS
> ....include what you already had...
> ### Android
> ...remove the parameter that was added or set to false...